src/order-builder/helpers.ts

```diff
@@ -140,18 +152,13 @@ export const createOrder = async (
     return buildOrder(eoaSigner, clobContracts.Exchange, chainId, orderData);
 };
 
-/**
- * Translate simple user market order to args used to generate Orders
- */
-export const buildMarketBuyOrderCreationArgs = async (
-    signer: string,
-    maker: string,
-    signatureType: SignatureType,
-    userMarketOrder: UserMarketOrder,
-): Promise<OrderData> => {
+export const getMarketOrderRawAmounts = (
```

Comment:
this is still only for market buys right?

Reply:
yep, exactly. I just changed its name to be more clear.